tests/command_default_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandDefault(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	t.Run("Command should not be empty", func(t *testing.T) {
```

Comment:
Shall we add more test cases here?
Like assert number of command matches the number defined in `DiceCmds = map[string]DiceCmdMeta{}`
Also few negative scenarios as part of unit tests.

Reply:
What do you mean by negative scenarios in this context? Verify the length of each command output?